en-US/emails/2017/copyright.lang

```diff
@@ -0,0 +1,81 @@
+## NOTE: Email to remind people we’re doing the campaign and ask them to share their story
+
+
+# Subject candidate 1
+;Share your story —
+Share your story —
```

Comment:
Maybe I should duplicate the preheader and have one starting with — and one without, but that’s a bit overkill